UnityProject/Assets/Scripts/UI/Systems/AdminTools/PlayerAlerts.cs

```diff
@@ -19,6 +20,16 @@ public class PlayerAlerts : MonoBehaviour
 
 		private readonly List<PlayerAlertData> clientPlayerAlerts = new List<PlayerAlertData>();
 
+		private void Awake()
```

Comment:
I'd say it is better to just use our implementation of singleton monobehaviours

Reply:
done